cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -809,23 +825,30 @@ def find_job_id_parameters(self, job_id):
         else:
             return json.load(open(scratch_dir_parameters[0]))
 
-
     # TODO make sure that the list of parameters to ignore in the frontend is synchronized
     def generate_products_url_from_par_dict(self, products_url, par_dict) -> str:
         par_dict = par_dict.copy()
+
+        if ('scw_list' in par_dict and self.use_scws is not None) or self.use_scws == 'no':
+            # for the frontend
+            par_dict['use_scws'] = self.use_scws
```

Comment:
will this not create cases when `use_scws` is not set in the url?
I guess it does not matter sometimes, but it does not hurt to have it always set for the frontend.

Reply:
In case `use_scws` is not set:

- If `scw_list` is set, then `use_scws` is set to automatically `form_list`
- If `scw_list` is not set, then `use_scws` remains not set, since it won't be needed

Or am I miss some case?

Reply:
if `scw_list` is not set, frontend can still benefit from `use_scws`. Else it takes the default which is fine sometimes, but why not set always?

And for file, yes, another case.